src/types/chains.types.ts

```diff
@@ -1,6 +1,3 @@
-enum ChainType {
-  Parachain = 'parachain',
-  RelayChain = 'relayChain'
-}
+type XcmChains = 'polkadot' | 'interlay';
```

Comment:
Being XCM an acronym, you should name the type `XCMChains`.

Take for instance when we use variable with `USD`, we dont write `AmountUsd`. It becomes hard to read.

Reply:
Good catch :) thanks